exact_cover/wrapper.py

```diff
@@ -1,10 +1,14 @@
 from exact_cover_impl import get_exact_cover as raw_get_exact_cover
 
 from .error import NoSolution
+from .io import DTYPE_FOR_ARRAY
 
 
 def get_exact_cover(matrix):
-    result = raw_get_exact_cover(matrix)
+    if matrix.dtype == DTYPE_FOR_ARRAY:
```

Comment:
This is great. We should add a Python unit test of this functionality, eg the same simple exact cover problem with arrays of int32, bool, and explicit DTYPE_FOR_ARRAY.

Reply:
Test added!